src/Host/Broker/Impl/Sessions/SessionsController.cs

```diff
@@ -35,16 +35,8 @@ public class SessionsController : Controller {
                 return Task.FromResult<IActionResult>(new ApiErrorResult(BrokerApiError.NoRInterpreters));
             }
 
-            SecureString securePassword = null;
-            string password = User.FindFirst(Claims.Password)?.Value;
-            if (password != null) {
-                securePassword = new SecureString();
-                foreach (var ch in password) {
-                    securePassword.AppendChar(ch);
-                }
-            }
-
             string profilePath = User.FindFirst(Claims.RUserProfileDir)?.Value;
+            var password = User.FindFirst(Claims.Password)?.Value.ToSecureString();
```

Comment:
Cant this be stored in Cred Manager instead of having this floating around? we can CredRead as needed.